### PR TITLE
Update CoinApi symbol mapping + symbol properties database

### PR DESCRIPTION
**Pull request by StefanoRaggi**
#### Description
- Updated symbol properties database for `GDAX`, `Bitfinex`, `Binance`
- Updated symbol mappings in `CoinApiSymbolMapper`

#### Related Issue
Includes #4708, #4741, #4745, #4785 

#### Motivation and Context
- Support for new pairs on Crypto brokerages

#### Requires Documentation Change
No.

#### How Has This Been Tested?
- Updated unit tests

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which improves implementation)
- [ ] Performance (non-breaking change which improves performance. Please add associated performance test and results)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [ ] I have added tests to cover my changes. 
- [ ] All new and existing tests passed.
- [ ] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

File: ToolBox/CoinApi/CoinApiSymbol.cs [hunk 1]
```diff
@@ -33,5 +33,15 @@ public class CoinApiSymbol
 
         [JsonProperty("asset_id_quote")]
         public string AssetIdQuote { get; set; }
+
+        [JsonProperty("price")]
+        public decimal Price { get; set; }
+
+        public bool IsActive => Price > 0;
```
Comment (Martin-Molinero) [inline ToolBox/CoinApi/CoinApiSymbol.cs:40]: Think `Price` & `IsActive` aren't used anymore after the last commit?

**Review by Martin-Molinero — APPROVED** [review]
Looks good 👍 leaving minor find

File: ToolBox/CoinApi/CoinApiSymbolMapper.cs [hunk 2]
```diff
@@ -199,7 +199,10 @@ private void LoadSymbolMap(string[] exchangeIds)
             // <Exchange>_SPOT_<BaseCurrency>_<QuoteCurrency>_<ExtraSuffix>
             // Those cases should be ignored for SPOT prices.
             _symbolMap = result
-                .Where(x => x.IsActive && x.SymbolType == "SPOT" && x.SymbolId.Split('_').Length == 4)
+                .Where(x => x.SymbolType == "SPOT" &&
+                    x.SymbolId.Split('_').Length == 4 &&
+                    // exclude Bitfinex BCH pre-2018-fork as for now we don't have historical mapping data
+                    (x.ExchangeId != "BITFINEX" || x.AssetIdBase != "BCH" && x.AssetIdQuote != "BCH"))
```
Comment (Martin-Molinero) [inline ToolBox/CoinApi/CoinApiSymbolMapper.cs:205]: Was wondering what's the effect of excluding `BCH` here?

Comment (StefanoRaggi) [reply ToolBox/CoinApi/CoinApiSymbolMapper.cs:205]: The CoinApi tickers for BCH currently in the symbol list are:
- BCH - Bitcoin Cash before the Nov2018 fork (not active)
- BCHABC - Bitcoin Cash ABC (renamed) (active)
- BCHSV - Bitcoin Cash SV (fork) (active)

The Bitfinex tickers for BCH are:
- BAB - Bitcoin Cash ABC
- BSV - Bitcoin Cash SV

The Lean tickers for BCH ("bitfinex" market) are:
- BCH - Bitcoin Cash ABC
- BSV - Bitcoin Cash SV

This BCH exclusion is required after this missing mapping was added:
https://github.com/QuantConnect/Lean/pull/4888/files#diff-2cb47206de766f5b1077db5bb591cd81b51c350e6a52a8630570da460c3a2f9bR60

**Review by jaredbroad — APPROVED** [review]
😍 beauty nice work @StefanoRaggi

**Pull request merged**